package/src/common/Input/DateInput.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useContext, useState } from "react";
+import BaseInput from "./BaseInput/BaseInput";
+import { InputFacadeProps } from "./BaseInput/BaseInput.models";
+import iconCalendar from "../../icons/date-input.png";
+import { NavContext } from "../../NavContext";
+import DatePicker, { DateType } from "../Input/DatePicker";
+import DateModule from "../Input/DatePicker/DateModule";
+
+const DateInput = React.forwardRef<HTMLDivElement, InputFacadeProps>((props, ref) => {
+  const { backScreen, nextScreen } = useContext(NavContext);
+
+  const [inputNotSupported] = useState(!isTypeSupported("date"));
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.currentTarget.value;
+    const date = {
+      year: value.split("-")[0],
+      month: value.split("-")[1],
+      day: value.split("-")[2],
+    };
+
+    props.onChange?.(e.currentTarget.name, date, e.currentTarget.type);
+  }
+
+  const formatValue = (value?: any) => {
```

Comment:
Is this kind of formatting used in multiple places? Maybe it would be a good idea to have a separate DateFormatter which can be extended in the future for multiple date display styles